### PR TITLE
Fix delete VCH error event handling

### LINKED ISSUE #128: If VCH deletion returns an error, the user is not given any indication that a failure has occurred
**VIC version:**

`vic-v1.3.0-rc2-2628-9e2a95b6.ova`

**Deployment details:**

vSphere Client version 6.5.0.10000

**Steps to reproduce:**

Delete a VCH in a bad state, or for which deletion will fail for some other reason.

**Actual behavior:**

A 400 is returned by the user, but nothing is shown to the user.

**Expected behavior:**

The fact that an error has occurred is shown the user, along with any error message.

**Logs:**

`vic-machine-server.log`:
```
Nov 28 2017 23:04:18.120Z INFO  op=1.15: Request: DELETE /container/target/sc-rdops-vm16-dhcp-241-196.eng.vmware.com/vch/vm-820 HTTP/1.1
Nov 28 2017 23:04:18.169Z INFO  op=1.15: Validating target
Nov 28 2017 23:04:18.371Z INFO  op=1.15: Response: 400 Bad Request
```

`vsphere_client_virgo.log`:
```
[2017-11-28T23:04:17.660Z] [WARN ] serenity-thread-pool-4       70000128 100006 200001 com.vmware.vise.util.session.SessionUtil                          Just clobbered an existing httpRequest 
[2017-11-28T23:04:17.661Z] [WARN ] serenity-thread-pool-4       70000128 100006 200001 com.vmware.vise.util.session.SessionUtil                          Just clobbered an existing httpSession 
```

### PR DESCRIPTION
**Pull request by cristianfalcone**
Also, this PR fixes delete VCH button style

Fixes #128

### CODE REVIEW

File: h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.template.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
 
   <clr-dg-row *ngFor="let item of vchs">
     <clr-dg-action-overflow class="{{item.name}}">
-      <button class="action-item" (click)="deleteVch(item)" class="{{item.name}}">Delete</button>
+      <button class="action-item {{item.name}}" (click)="deleteVch(item)">Delete</button>
```
Comment (jak-atx) [inline h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.template.html]: I'm not quite clear why we are using the vch name as a class in the first place?

Comment (cristianfalcone) [reply h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.template.html]: I think you or jkim added it for e2e tests purposes.

Comment (cristianfalcone) [reply h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.template.html]: https://github.com/vmware/vic-ui/blob/master/h5c/vic/src/vic-webapp/e2e/app.po.ts#L139

Comment (jak-atx) [reply h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.template.html]: Oh right. Ha. thanks.

**Pull request merged**